.github/workflows/auto-merge-dependabot.yml

```diff
@@ -0,0 +1,66 @@
+                  steps.metadata.outputs.dependency-names == '@babel/cli' ||
+                  steps.metadata.outputs.dependency-names == '@babel/core' ||
+                  steps.metadata.outputs.dependency-names == '@babel/plugin-proposal-class-properties' ||
+                  steps.metadata.outputs.dependency-names == '@babel/plugin-syntax-jsx' ||
+                  steps.metadata.outputs.dependency-names == '@babel/polyfill' ||
+                  steps.metadata.outputs.dependency-names == '@types/classnames' ||
+                  steps.metadata.outputs.dependency-names == '@types/dinero.js' ||
+                  steps.metadata.outputs.dependency-names == '@types/dompurify' ||
+                  steps.metadata.outputs.dependency-names == '@types/gtag.js' ||
+                  steps.metadata.outputs.dependency-names == '@types/jest' ||
+                  steps.metadata.outputs.dependency-names == '@types/jest-environment-puppeteer' ||
+                  steps.metadata.outputs.dependency-names == '@types/jquery' ||
+                  steps.metadata.outputs.dependency-names == '@types/lodash' ||
+                  steps.metadata.outputs.dependency-names == '@types/puppeteer' ||
+                  steps.metadata.outputs.dependency-names == '@types/react' ||
+                  steps.metadata.outputs.dependency-names == '@types/react-dom' ||
+                  steps.metadata.outputs.dependency-names == '@types/wordpress__block-editor' ||
+                  steps.metadata.outputs.dependency-names == '@types/wordpress__blocks' ||
+                  steps.metadata.outputs.dependency-names == '@types/wordpress__data' ||
+                  steps.metadata.outputs.dependency-names == '@types/wordpress__data-controls' ||
+                  steps.metadata.outputs.dependency-names == '@types/wordpress__editor' ||
+                  steps.metadata.outputs.dependency-names == '@types/wordpress__notices' ||
+                  steps.metadata.outputs.dependency-names == '@typescript-eslint/eslint-plugin' ||
+                  steps.metadata.outputs.dependency-names == '@typescript-eslint/parser' ||
+                  steps.metadata.outputs.dependency-names == 'chalk' ||
+                  steps.metadata.outputs.dependency-names == 'circular-dependency-plugin' ||
+                  steps.metadata.outputs.dependency-names == 'commander' ||
+                  steps.metadata.outputs.dependency-names == 'copy-webpack-plugin' ||
+                  steps.metadata.outputs.dependency-names == 'eslint-import-resolver-typescript' ||
+                  steps.metadata.outputs.dependency-names == 'gh-pages' ||
+                  steps.metadata.outputs.dependency-names == 'markdown-it' ||
+                  steps.metadata.outputs.dependency-names == 'promptly' ||
+                  steps.metadata.outputs.dependency-names == 'react-docgen' ||
+                  steps.metadata.outputs.dependency-names == 'wp-types'
+              }}"
```

Comment:
I'm wondering if it would be possible to use something like:
```
contains(fromJson('["promptly", "react-docgen", "wp-types"]'), steps.metadata.outputs.dependency-names)
```
If possible, it would be probably easier to maintain in the future, what do you think?

Reply:
Hey @thealexandrelara ! That looks promising: I'll take a look and open a separate PR to update it; thanks for the suggestion 🙌 !